state.go

```diff
@@ -356,10 +379,54 @@ func (i *InitState) Handle(ctx *StateContext, c Controller) (State, bool) {
 
 	log.Infof("handling loaded state: %s", sd.Name)
 
-	// chack last known state
+	i.SetTransition(sd.LeaveTransition) // Handle an unfinished leave transition
+
+	// check last known state
 	switch sd.Name {
```

Comment:
Should we have some default here as well? I am thinking now, that maybe we will end up with the situation that after reboot (running with the new update) we will have some state that was not supported (by previous client).